db.js

```diff
@@ -1,18 +1,16 @@
-// const mongoose = require('mongoose');
-
 const mongodb = require('mongodb');
 const MongoClient = mongodb.MongoClient;
 
-//Set up default mongoose connection
-var mongoDBLocal = 'mongodb://localhost:27017'
-var mongoDB = 'mongodb+srv://root:supersecret@cluster0.ddfbs.mongodb.net/softball'
-// mongoose.connect(mongoDB || mongoDBLocal, { useNewUrlParser: true, useUnifiedTopology: true })
-
-//Get the default connection
-// var db = mongoose.connection;
+var mongoDB = 'mongodb://localhost:27017'
 
-//Bind connection to error event (to get notification of connection errors)
-// db.on('error', console.error.bind(console, 'MongoDB connection error:'));
+if (process.env.NODE_ENV === 'production') {
+    console.log("It's in production" );
+    // 
+    mongoDB = 'mongodb+srv://root:supersecret@cluster0.ddfbs.mongodb.net/softball'
```

Comment:
Since we've specified which `db` we use in the place where we use the db, we don't need to put `/softball` in this url. Please remove it.  (it's working fine, though)